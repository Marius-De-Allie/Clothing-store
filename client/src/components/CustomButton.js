import React from 'react';

// styles.
import '../styles/customButton.scss';

const CustomButton = ({ children, isGoogleSignIn, inverted, ...otherProps}) => {
    console.log(inverted);
    return (
        <button 
            className={`${inverted ? 'inverted' : ''} ${isGoogleSignIn ? 'google-sign-in' : ''} custom-button`} 
            {...otherProps}
        >
            {children}
        </button>
    );
};

export default CustomButton;

