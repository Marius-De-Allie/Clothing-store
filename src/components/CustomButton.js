import React from 'react';

// styles.
import '../styles/customButton.scss';

const CustomButton = ({ children, ...otherProps}) => (
    
    <button className="custom-button" {...otherProps}>
        {children}
    </button>
);

export default CustomButton;