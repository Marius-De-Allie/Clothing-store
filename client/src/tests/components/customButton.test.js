import React from 'react';
import { shallow } from 'enzyme';
import CustomButton from '../../components/CustomButton';

it('CustomButton component renders properly', () => {
    const wrapper = shallow(<CustomButton />);
    expect(wrapper).toMatchSnapshot();
});