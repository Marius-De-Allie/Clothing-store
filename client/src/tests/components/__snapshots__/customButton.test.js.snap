// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomButton component renders properly 1`] = `ShallowWrapper {}`;
